csharp_dotnetcore/5.MultiTurn-Prompts-Bot/BotAccessors.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs;
+
+namespace MultiTurn_Prompts_Bot
+{
+    public class BotAccessors
```

Comment:
>BotAccessors [](start = 17, length = 12)

Same deal with constructor-giving-the-state object (see example 2 in core), and explicitly saving on state object before sending response so we set a conservative example.